### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by osamaashraf6**
This is my portfolio which expresses what I do in it the followings 👎 

- Add the HTML for the structure.
- Add the CSS file for the styling.
- Use the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.
- Use Figma a collaborative UI design tool built in the browser. 
- Use Flexbox a layout tool in CSS that helps you easily position HTML elements on the page.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    <!-- end fonts -->
+    <link rel="stylesheet" href="../static/css/index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start nav -->
+    <nav class="nav" id="nav">
+      <div class="container">
+        <div class="nav__container__parnav">
+          <div class="nav__container__parnav__brand">
```
Comment (awais-amjed) [inline templates/index.html]: Kindly replace the `nav` tag with the `header` tag for semantic reasons 🤗
- Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: templates/index.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          <div class="nav__container__parnav__bars">
+            <div class="nav__container__parnav__bars__responsive">
+              <img
+                src="..//static/imgs/Union@2x.png"
+                alt="Union@2x"
+                class="responsive nav__container__parnav__bars__responsive__respo"
+              />
+            </div>
+          </div>
+        </div>
```
Comment (awais-amjed) [inline templates/index.html]: After removing the `nav` tag from above, wrap the hamburger in the `nav` tag for semantic reasons described above 🤗

File: templates/index.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    <link rel="stylesheet" href="../static/css/index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start nav -->
+    <nav class="nav" id="nav">
+      <div class="container">
+        <div class="nav__container__parnav">
+          <div class="nav__container__parnav__brand">
+            <a href="#">My Logo</a>
```
Comment (awais-amjed) [inline templates/index.html:24]: Kindly update the `font-size` and `color` of logo to match the Figma design 🤗

Your design:

image

Figma design:

image

File: templates/index.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    </nav>
+    <!-- end nav -->
+    <!-- start sectionone -->
+    <section class="sectionone" id="sectionone">
+      <div class="overlay">
+        <div class="sectionone__heroimage">
+          <h2>
+            I’m Alexandra <br />
+            Glad to see you!
+          </h2>
```
Comment (awais-amjed) [inline templates/index.html]: Kindly replace the `h2` tag with `h1` since this is the main heading of our page 🤗

File: templates/index.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+              />
+            </div>
+            <div class="social__responsive">
+              <img
+                src="../static/imgs/angellist.png"
+                alt="angellist"
+                class="social__responsive__respo"
+              />
+            </div>
+          </div>
```
Comment (awais-amjed) [inline templates/index.html]: Kindly note that this list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) 🤗
- _Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href._

File: templates/index.html [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+            Glad to see you!
+          </h2>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through <br />
+            some of my work and experience! If you like what you see and have a
+            project you need <br />
+            coded, don’t hestiate to contact me.
+          </p>
+          <span>LET’S CONNECT</span>
```
Comment (awais-amjed) [inline templates/index.html]: Kindly note that this span should be a `button` or `a` tag 🤗
- _Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href._

File: templates/index.html (hunk 4 above)
Comment (awais-amjed) [inline templates/index.html]: Kindly make sure to use the font-size and color described in Figma to match the design 🤗
- Font size should be bigger
- Font color should be `172B4D`

Your design:

image

Figma design:

image

File: templates/index.html [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+            I’m Alexandra <br />
+            Glad to see you!
+          </h2>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through <br />
+            some of my work and experience! If you like what you see and have a
+            project you need <br />
+            coded, don’t hestiate to contact me.
+          </p>
```
Comment (awais-amjed) [inline templates/index.html]: Kindly fix the font color of this text also to match the Figma design, it should be `344563` 🤗

File: templates/index.html (hunk 6 above)
Comment (awais-amjed) [inline templates/index.html]: Kindly fix the font color of this button too, it should be `7F8CFF` 🤗

File: templates/index.html [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+                class="responsive nav__container__parnav__bars__responsive__respo"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <!-- end nav -->
+    <!-- start sectionone -->
+    <section class="sectionone" id="sectionone">
```
Comment (awais-amjed) [inline templates/index.html:43]: Kindly also try to fix the background image of this section to match the Figma design 🤗

Your design:

image

Figma design:

image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @osamaashraf6 , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter issues ✔️ 
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- It's better to keep the `index.html` file in the base folder of the project, instead of moving it to a `templates` folder, since later on when you deploy the project, GitHub Pages won't be able to locate it 😃

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
# Approved ✔
### Hi @osamaashraf6,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by frank1738 — COMMENTED** [review]
Hi @osamaashraf6 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has already been approved

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: templates/index.html (hunk 1 above)
Comment (osamaashraf6) [reply templates/index.html]: - Thanks 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊

File: templates/index.html (hunk 8 above)
Comment (osamaashraf6) [reply templates/index.html:43]: - Thanks 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊

**Comment by osamaashraf6** [comment]
- Thanks 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊

**Pull request merged**